azkaban-web-server/src/main/java/cloudflow/docs/ExecutionDetailSample.md

```diff
@@ -0,0 +1,202 @@
+{
+  "executionId": 3,
+  "submitUser": "azkaban",
+  "submitTime": 1581552354517,
+  "concurrentOption": "skip",
+  "failureAction": "FINISH_CURRENTLY_RUNNING",
+  "experimentId": "none",
+  "description": "none",
+  "previousFlowExecutionId": "none",
+  "notifyFailureFirst": false,
+  "notifyFailureLast": false,
+  "rootFlow": {
+    "nodeId": "flowPriority2",
```

Comment:
Can you remove `nodeId` property?

Reply:
To clarify - you mean to remove it only from flow and embedded flows sections as it will be captured in the `flowInfo` for those ?
For jobs this information is not captured anywhere else in the output.